### PR TITLE
Elaborate on managing Conda python, channels

### PR DESCRIPTION
**Pull request by muyuxuan946**
This PR solves the aspects mentioned in Issue #484: Development Process: Introduction for the Conda #484. It added instructions for five major types of Conda tasks including Managing Python, Managing virtual packages, Creating custom channels, Creating projects, and Viewing command-line help. The instruction introduces the commonly used commands for these tasks.

### CODE REVIEW

**Review by emma0925 — CHANGES_REQUESTED** [review]
Hi Yuxuan, 
Your writing is very detail and clear! 
One suggestion I have is to be consistent with the file structure of pr #604, I would suggest you to break each sub-section into different files. 
Also, please resolve the conflict with setup_conda.md in the main branch. I think by break all the sections down to different files. The conflict will be resolved.

File: Topics/Development_Process/Conda/Setup_Conda.md [hunk 1]
```diff
@@ -117,8 +117,135 @@ In this prompt, type in `conda init powershell`, and press enter. It should show
...
+
+![conda_update](conda_update2.png)
+
+## Managing Vitual Packages 
+
+"Virtual" packages are injected into the conda solver to allow real packages to depend on features present on the system that cannot be managed directly by conda, like system driver versions or CPU features. Virtual packages are not real packages and not displayed by conda list. Instead conda runs a small bit of code to detect the presence or absence of the system feature that corresponds to the package. The currently supported list of virtual packages includes:
+
+![conda_ver_list](conda_ver_list.png)
+
+### Listed detected Visual Packages
```
Comment (YankeMao) [inline Topics/Development_Process/Conda/Setup_Conda.md:212]: I think here the sub-title should be virtual instead of visual.

**Comment by emma0925** [comment]
Hi Yuxuan, I fogot to add citations for my pr #600, it's already merged. Could you add the citation section for me.
## References

- Conda Official Documentation. Available at: [Conda Documentation](https://docs.conda.io/projects/conda/en/latest/)
- Existing work and examples can be found in the Conda README directory within the repository. For specific examples and configurations, refer to the [Conda README](./README.md) in this repository.

**Comment by muyuxuan946** [comment]
will do that right now. Thank you for feedback.

File: Topics/Development_Process/Conda/Creating_projects.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+## Creating Pojects
```
Comment (emma0925) [inline Topics/Development_Process/Conda/Creating_projects.md:1]: change it to '#', so it matches the format of other files in the directory

File: Topics/Development_Process/Conda/Creating_projects.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+## Creating Pojects 
+Conda provides a convenient way to manage dependencies and environments for your projects. A project in Conda is a directory that contains your project code along with a configuration file (environment.yml) specifying the dependencies required to run the project.
+
+### Creating the Project's file
```
Comment (emma0925) [inline Topics/Development_Process/Conda/Creating_projects.md:4]: change it to ##

Comment (emma0925) [reply Topics/Development_Process/Conda/Creating_projects.md:4]: and do the same thing for all of your files.

**Comment by YankeMao** [comment]
I'd suggest to edit the reference list to keep consist with the other documents for Conda in the repo

**Comment by YankeMao** [comment]
Also, the reference list is needed for each .md file for the consistent aim.

**Review by YankeMao — APPROVED** [review]
Good job! Approved.

**Review by emma0925 — APPROVED** [review]
LGTM

**Pull request merged**